backend/.docker/app-prod/php-fpm.sh

```diff
@@ -50,9 +50,9 @@ fi
 if [ "${CONSUME_MESSAGES:-}" = true ]; then
     echo 'Consume messages...'
 
-    console messenger:consume --time-limit=3600 -vv >&1
+    console messenger:consume async --time-limit=3600 -vv >&1
 
-    exit 0;
+    exit 0
```

Comment:
Can you replace the other exiit on this file? I know they are not related with your PR, but for consistency

Reply:
allright!

Reply:
@jordisala1991 changed :)